blog/_src/posts/2017-11-16-monotonicity-types-towards-a-type-system-for-eventual-consistency.md

```diff
@@ -0,0 +1,97 @@
+    Title: Monotonicity Types: Towards A Type System for Eventual Consistency
+    Date: 2017-10-22T11:59:06
+    Tags: types, monotonicity, CRDTs, eventual consistency, by Kevin Clancy
+
```

Comment:
This line needs to say:

```
<!-- more -->

```

To tell frog "this post has no abstract". Otherwise, frog renders the whole post on the landing page:
http://prl.ccs.neu.edu/blog/index.html